### PR TITLE
Fix: How-to block preview console issues 

### PR DESCRIPTION
**Pull request by supriyajjadhav**
### Description
if i just deactivate infobox block and then hover How to block from left sidebar then this error fired. 
Issue - https://www.awesomescreenshot.com/image/20889574?key=5f63a936b4fe42d1a045e2d6f7d51c58 

### Types of changes
non-breaking change which fixes an issue

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/how-to/block.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ registerBlockType( 'uagb/how-to', {
 	example: {
 		innerBlocks: [
 			{
-				name: 'uagb/info-box',
+				name: 'uagb/how-to-step',
 				attributes: {
 					infoBoxTitle: 'Step 1',
```
Comment (RajkiranBagal) [inline src/blocks/how-to/block.js:32]: Need to also update attributes?

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/blocks/how-to/block.js (hunk 1 above)
Comment (supriyajjadhav) [reply src/blocks/how-to/block.js]: yes, updated

**Review by RajkiranBagal — APPROVED** [review]

**Comment by akshaylakh** [comment]
Console error -  [img](https://share.bsf.io/z8ukkNPv)

**Comment by supriyajjadhav** [comment]
> Console error - [img](https://share.bsf.io/z8ukkNPv)
@akshaylakh Added a separate [card ](https://app.startinfinity.com/b/Mi5zu3VRVzk/5s2b2cSP7a5/20482254-3031-4950-99db-4ca592613ee9?open=true&view=0f0c95d5-d336-4bdd-bb8b-f7cfb212b3d1&t=attributes&redirect=true) for this issue.

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**